plugins/uefi-capsule/fwupdate.md

```diff
@@ -0,0 +1,37 @@
+% fwupdate(1) @PACKAGE_VERSION@ | fwupdate man page
+
+NAME
+====
+
+**fwupdate** — debugging utility for UEFI firmware updates
+
+SYNOPSIS
+========
+
+| **fwupdate** \[CMD]
+
+DESCRIPTION
+===========
+
+**fwupdate** is a deprecated tool that allows deploying capsule updates.
```

Comment:
Except it's also used for debugging uefi-capsule problems.  It's the only way that we turn on/off logging by default today.

Reply:
Ya, I'm not 100% sure it's a good idea to rely on it -- the setting and clearing the variable is a bit janky and interacts badly if fwupd is deleting variables too. Maybe we should have something like `Debugging = true` in uefi_capsule.conf and handle it in the daemon?

Reply:
Yeah let's do that and make this noinst